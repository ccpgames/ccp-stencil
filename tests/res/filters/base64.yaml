apiVersion: v1
kind: Secret
metadata:
  name: {{app.slug}}-secrets
  namespace: {{app.namespace}}
  labels:
    version: {{app.version}}
type: Opaque
data:
  {% for k, v in app.secrets.items() %}
  {{k}}: "{{v|base64}}"
  {% endfor %}
