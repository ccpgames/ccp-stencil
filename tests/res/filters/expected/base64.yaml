apiVersion: v1
kind: Secret
metadata:
  name: foo-secrets
  namespace: bar-test
  labels:
    version: 1.2.3-beta.4
type: Opaque
data:
  this_is_confidential: "SXRzIGEgc2VjcmV0IHRvIEVWRVJZQk9EWSE="
  DB_PASSWORD: "UmVhbGx5IEJhZCBQYXNzd29yZCE="
